code/modules/lighting/lighting_turf.dm

```diff
@@ -89,7 +89,7 @@
 		else
 			lighting_clear_overlay()
 
-/turf/proc/get_corners(var/dir)
+/turf/proc/get_corners()
```

Comment:
is this overridden anywhere (or required to be)? because this could probably be a macro, ```T.get_corners()``` -> ```GET_CORNERS(T)```